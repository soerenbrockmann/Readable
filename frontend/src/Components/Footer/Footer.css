.footer {
  display: flex;
  justify-content: center;
  align-items: flex-start;
  border-top: 1px solid #e0e0e0;
}
