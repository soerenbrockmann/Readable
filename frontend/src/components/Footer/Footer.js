import React from "react";
import "./Footer.css";

const Footer = () => {
  return <div className="footer">{new Date().getFullYear()}</div>;
};

export default Footer;
