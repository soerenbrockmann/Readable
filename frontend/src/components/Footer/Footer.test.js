import React from "react";
import { shallow } from "enzyme";
import Footer from "./Footer";

describe("<Footer />", () => {
  const footer = shallow(<Footer />);
  const currentYear = new Date().getFullYear().toString();

  it("renders the current year", () => {
    expect(footer.text()).toBe(currentYear);
  });

  it("renders properly", () => {
    expect(footer).toMatchSnapshot();
  });
});
