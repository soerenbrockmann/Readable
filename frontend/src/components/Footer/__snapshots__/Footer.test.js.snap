// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> renders properly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Footer />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": 2018,
      "className": "footer",
    },
    "ref": null,
    "rendered": 2018,
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": 2018,
        "className": "footer",
      },
      "ref": null,
      "rendered": 2018,
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
